R/finalize-version.R

```diff
@@ -11,46 +10,29 @@ finalize_version_impl <- function(push, suggest_finalize = TRUE) {
   tag <- tag_version(force)
   #' 1. Force-pushes the created tag to the `"origin"` remote, if `push = TRUE`.
   if (push) {
-    push_tag(tag)
-    push_head(head)
+    cli_alert("Force-pushing tag {.field {tag}}.")
+    gert::git_tag_push(tag, force = force)
+    gert::git_push()
   } else if (suggest_finalize) {
     edit_news()
 
-    if (has_remote_branch(head)) {
-      command <- "fledge::finalize_version(push = TRUE)"
-    } else {
-      command <- "fledge::finalize_version()"
-    }
+    command <- "fledge::finalize_version(push = TRUE)"
 
     cli_alert_warning("Call {.code {command}}.")
     send_to_console(command)
   }
 }
 
-get_head_branch <- function() {
-  head <- git2r::repository_head()
-  stopifnot(git2r::is_branch(head))
-  head
-}
-
-push_tag <- function(tag) {
```

Comment:
I rather like the verbosity here. Is gert showing output by itself? Either way, I don't mind keeping our extra infos here.

Reply:
gert is quite chatty but IDK on top of my head what happens in these particular cases.

Reply:
Yeah, we'll see with galley. Should merge #74 and #70 first.